doc/appendix-de-kassensichv/reference-tables/reference-tables.md

```diff
@@ -279,3 +279,478 @@ For definitions regarding national laws, please refer to the appropriate appendi
+| `Z_KASSE_ID` | ID of the (closing) cashpoint | String (50) | `ftCashBoxIdentification` |
+| `Z_ERSTELLUNG` | Date of the cashpoint closing | ISO 8601 und RFC3339 date | From `ftReceiptMoment` of the daily closing receipt |
+| `Z_NR` | Nr. of the cashpoint closing | Integer | Automatically created and filled by ft |
+| `BON_ID` | Action-ID | String (40) | `cbReceiptReference` |
+| `BON_NR` | Receipt number | Long | `ftReceiptIdentification` (tbd: ftReceiptIdentification is a string, but number/long is needed) |
+| `BON_TYP` | Receipt type / action type| String (30) | Deducted from `ftReceiptCase` |
+| `BON_NAME` | Additional description related to the `BON_TYP` | String (60) | Mandatory if `BON_TYPE` is "AVSonstige", otherwise optional, can be sent via `ftReceiptCaseData` in JSON format. To send, add the key value pair `ReceiptName ` e.g. `"ftReceiptCaseData":"{ ..., "ReceiptName":"Sonstige Sonderwurst", ... }"` |
+| `TERMINAL_ID` | Mandatory, ID of the terminal that was used to record this receipt | String (50) | `cbTerminalID` |
+| `BON_STORNO` | Cancellation indicator | String | Mandatory if your receipt is a reverse receipt that voids another, previous receipt. In this case mark your receipt with the `ftReceiptCaseFlag` `0x0000000000040000` and reference the receipt to be voided via `cbPreviousReceiptReference`. Ft will then set the `BON_STORNO` flag in the DSFinV-K export and will also add a corresponding reference in the file "**Bon_Referenzen (references.csv)**" (see below). |
+| `BON_START` | Time of the action start | ISO 8601 and RFC3339 date | The action start can be within this cashpoint or outside of this cashpoint. If outside (e.g. by another system or another cashpoint) than it has to be provided in via `ftReceiptCaseData` in JSON format by adding the key value pair `ActionStartMoment`. E.g. `"ftReceiptCaseData":"{ ..., "ActionStartMoment":"2020-09-27T17:00:01", ... }"`. If not provided, ft tries to find the action start by following the `cbPreviousReceiptReference` path into the past until no more previous receipt references exist. ft will than fill this field with the value from `ftReceiptMoment` of the oldest receipt found in that chain. |
+| `BON_ENDE` | Time of the action end | String | `ftReceiptMoment` |
+| `BEDIENER_ID` | User-ID | String (50) | Mandatory, send via `cbUser` in JSON format by adding the key value pair `UserId` e.g. `"cbUser":"{ "UserId":"19292", "UserName":"Peter Lux"}"`|
+| `BEDIENER_NAME` | User name | String (50) | Mandatory, send via `cbUser` in JSON format by adding the key value pair `UserName` e.g. `"cbUser":"{ "UserId":"19292", "UserName":"Peter Lux"}"`|
+| `UMS_BRUTTO` | Gross total turnover | Decimal (2) | Automatically filled by ft |
+| `KUNDE_NAME` | Name of beneficiary customer | String (50) | Mandatory if not exempted in relation to § 148 AO. Send via `cbCustomer` in JSON format by adding the key value pair `CustomerName` e.g. `"cbCustomer":"{"CustomerName":"Max Wanne",...}"`|
```

Comment:
Same here.. also we could probably hash it. Additionally this could be a potential data problem if we store confident data for customers of our customers.

Reply:
done